micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/Login/login.js

```
@@ -145,77 +141,84 @@
   };
-  const defaultTenant = Digit.ULBService.getStateId();
+
   const defaultValue = {
-    code: defaultTenant,
-    name: Digit.Utils.locale.getTransformedLocale(`TENANT_TENANTS_${defaultTenant}`),
+    code: Digit?.ULBService?.getStateId(),
+    name: Digit.Utils.locale.getTransformedLocale(`TENANT_TENANTS_${Digit?.ULBService?.getStateId()}`),
   };
 
-  const config = [{ body: propsConfig?.inputs }];
+  let config = [{ body: propsConfig?.inputs }];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'config' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)